### ISSUE TITLE
Can not create a user profile in firefly - firefly does not support letter 'ä' in Windows 10 user name and user directory

### ISSUE THREAD

**Issue by Tombaaa** [iotaledger/firefly#995]
tried to create a profile in firefly, always get the error "something went wrong" in the right lower corner.
Error log is empty
![image](https://user-images.githubusercontent.com/77152394/115623892-1a32e980-a2fa-11eb-8256-1f86f32db2e1.png)

Diagnostic:
Firefly v1.0.0
Language: en
Currency: USD
Node settings: Automatic node selection
Platform: win32
Platform Version: 10.0.19042
Platform Architecture: x64
CPU Count: 8
Total Memory: 14269.2 MB
Free Memory: 7680.1 MB
User Path: C:\Users\Bänni\AppData\Roaming\Firefly

as you can see, I use Bänni as user account. Apparently it cause this error, if the user path contains a special character like "ä". 
When I change to another user account without a 'ä', I can create a profile.

**Comment by Grachas** [comment]
I think it similarly does not support "ğ" character as well. I have encountered this "Something Went Wrong" issue too with error log being empty.

**Comment by Tombaaa** [comment]
Firefly seems to be installed by default in the user directory of the signed in windows user and hence uses the windows user name. Tried to change the windows user name and the directory name and changed a ton of regedit entries to get rid of the 'ä', but ended up loosing my completed desktop. So I switched back and I now wait on a fix by IF.

**Comment by Tombaaa** [comment]
created a new issue with more details and screenshots. 
See issue No 1015
